src/hal/clock.c

```diff
@@ -2,7 +2,7 @@
 
 void sys_tick_handler(void) {
 
-  // TODO fill with cron calls
-
+  // FIXME: app dependent function
```

Comment:
Remove this comment

Reply:
This comment was to indicate that we are using in hal a reference to a function in app, which is not the best approach. 

It is probably a best solution to move sys_tick_handler to some file in the app folder. What do you think?

Reply:
We can address that in a future issue